contracts/avalanche/connectors/swap/helpers.sol

```diff
@@ -0,0 +1,102 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma abicoder v2;
+
+import { InstaConnectors } from "../../common/interfaces.sol";
+
+abstract contract Helper {
+	/**
+	 * @dev Instadapp Connectors Registry
+	 */
+	InstaConnectors internal constant instaConnectors =
+		InstaConnectors(0x127d8cD0E2b2E0366D522DeA53A787bfE9002C14);
+
+	struct InputData {
+		address buyAddr;
+		address sellAddr;
+		uint256 sellAmt;
+		uint256[] unitAmts;
+		bytes[] callDatas;
+		uint256 setId;
+	}
+}
```

Comment:
You can define them inside SwapHelpers as well.

Reply:
Done.